packages/main/src/i18n/messagebundle.properties

```diff
@@ -243,6 +243,9 @@ MULTIINPUT_SHOW_MORE_TOKENS={0} More
 #XTOL: Tooltip for panel expand title
 PANEL_ICON=Expand/Collapse
 
+#XACT: ARIA description for the segmented button
+SEGMENTEDBUTTON_ARIA_DESCRIPTION=Segmented button
```

Comment:
Is this really the right text? Isn't "Segmented button" a term only we know, and not meant for the end user?

Reply:
That's the specification given me by @elenastoyanovaa .
Eli, I think you can give more information?

Reply:
Yes, this is the right text. Since the component/control is specific we are adding an aria-roledescription according to Aria 1.1. This is already discussed and will also be implemented in openui5.